calculator_test.go

```diff
@@ -14,11 +23,90 @@ func TestAdd(t *testing.T) {
 	}
 }
 
-// func TestSubtract(t *testing.T) {
-// 	t.Parallel()
-// 	var want float64 = 2
-// 	got := calculator.Subtract(4, 2)
-// 	if want != got {
-// 		t.Errorf("want %f, got %f", want, got)
-// 	}
-// }
+func TestAdd1and1(t *testing.T) {
```

Comment:
We no longer really need this separate test if it's covered by TestAddStruct, do we?